### ISSUE TITLE
Document.onmousewheel event was removed from typescript

### ISSUE THREAD

**Issue by Slash7GNR** [Microsoft/TypeScript#30067]
**TypeScript Version:**  3.4.0-dev.201xxxxx

**Search Terms:**

**Code**

```ts
// A *self-contained* demonstration of the problem follows...
// Test this by running `tsc` on the command-line, rather than through another build tool such as Gulp, Webpack, etc.
```

**Expected behavior:**
document.onmousewheel should be existed.

**Actual behavior:**
Intellisense & on build getting an error: Property 'onmousewheel' does not exist on type 'Document'.

**Playground Link:** 

**Related Issues:** 
I've noticed that this event was removed on the following commit from lib.dom.d.ts:
https://github.com/Microsoft/TypeScript/commit/79e5e79ef7724066889fbce97e9e6d6db4e746b5#diff-46fd87925e4552c166ec188712741c3f

I was wondering if it's a mistake or if there was a reason for the removal.

**Comment by Slash7GNR** [comment]
It was transferred to the Window, my bad :)

**Comment by kungfooman** [comment]
Any work on this? It just creates useless/wrong errors:

![image](https://user-images.githubusercontent.com/5236548/126945542-dca6ca59-285e-49c2-b67a-cf1e46a7bd85.png)
